pyatlan/model/assets/process.py

```diff
@@ -104,6 +104,10 @@ def __setattr__(self, name, value):
     """
     TBC
     """
+    ALTERYX_WORKFLOW: ClassVar[RelationField] = RelationField("alteryxWorkflow")
```

Comment:
This is a one-off customization (in-progress experiment) — we should remove from generated code.